CHANGELOG.md.meta

```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
```

Comment:
Do we want to merge `CHANGELOG.md.meta` , `CODE_OF_CONDUCT.md.meta`, ` CONTRIBUTING.md.meta`?